framework_tvb/tvb/core/entities/storage/session_maker.py

```diff
@@ -53,10 +54,13 @@
 
 def build_db_engine():
     if TvbProfile.current.db.SELECTED_DB == 'postgres':
-        ### Control the pool size for PostgreSQL, otherwise we might end with multiple
-        ### concurrent Python processes failing because of too many opened connections.
-        DB_ENGINE = create_engine(TvbProfile.current.db.DB_URL, pool_recycle=5, max_overflow=1,
-                                  pool_size=TvbProfile.current.db.MAX_CONNECTIONS)
+        if TvbProfile.current.db.MAX_CONNECTIONS == 0:
+            DB_ENGINE = create_engine(TvbProfile.current.db.DB_URL, poolclass=NullPool)
```

Comment:
Also here pls add a comment, just to state that in this case, we are using a PGbouncer